pubspec.yaml

```diff
@@ -1,7 +1,9 @@
 name: phone_number
 description: Flutter plugin for parsing, formatting and validating international phone numbers.
-version: 0.6.2+4
+version: 0.6.3
 homepage: https://github.com/nashfive/phone_number
+authors:
+  - Onat Cipli <ciplionat35@gmail.com>
```

Comment:
Oh I forgot to update the authors.. 
Can you remove this for the moment? I will add the complete list in a commit.

Reply:
Okay I was going to add you but I couldn't find your email, I'm removing